gamemodes/terrortown/gamemode/client/cl_shop.lua

```diff
@@ -0,0 +1,25 @@
+---
+-- A class to handle all shop requirements
+-- @author ZenBre4ker
+-- @class Shop
+
+SHOP = SHOP or {}
```

Comment:
I'm not sure if this should really be `SHOP`, shouldn't it be `shop`? I thought we only use uppercase if there are instances of this, e.g. `TARGET_DATA`

I personally would put it in `libraries` as well

Reply:
you are right it should be `shop`.
But I dont think it should be a library as it will handle mostly our internal stuff and IMHO libraries should handle general reusable stuff.